### ISSUE TITLE
no-trailing-whitespace: config for JSDocs

### ISSUE THREAD

**Issue by buehler** [palantir/tslint#2167]
### Bug Report

- __TSLint version__: `4.4.2`
- __TypeScript version__: `2.1.5`
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
import { injectable } from 'inversify';
import { Disposable, ExtensionContext } from 'vscode';

/**
 * Base extension class. Does provide the basic structure of an extension.
 * 
 * @export
 * @abstract
 * @class BaseExtension
 * @implements {Disposable}
 */
@injectable()
export abstract class BaseExtension implements Disposable {
    /**
     * Creates an instance of BaseExtension.
     * 
     * @param {ExtensionContext} context
     * 
     * @memberOf BaseExtension
     */
    constructor(protected readonly context: ExtensionContext) { }

    /**
     * Initialize the extension. Should register disposables to the extension context
     * and register commands and such operations.
     * 
     * @abstract
     * 
     * @memberOf BaseExtension
     */
    public abstract initialize(): void;

    /**
     * Dispose the extension.
     * 
     * @abstract
     * 
     * @memberOf BaseExtension
     */
    public abstract dispose(): void;
}
```

with `tslint.json` configuration: default "recommended"

#### Actual behavior

It does mourn about the spaces in the jsdocs. VSCode does insert 1 space after each asterisk in the jsdocs.

#### Expected behavior

It would be cool if one could configure the `no-trailing-whitespace` rule, to ignore JSDocs and / or comments in general.

**Comment by ajafff** [comment]
The `ignore-comments` option was added in #2153 and will be shipped with the next release.
I will probably send a PR to add an `ignore-jsdoc` option in till the end of the week.

**Comment by ajafff** [comment]
Fixed by #2177
